README.md

```diff
@@ -576,37 +584,35 @@ well for you—and when it might not.
+  (Rachel) wonders whether we might choose one or the other for clarity within this Guide,
+  only marking here that you are likely to hear both.
+
 - 🄳 Vesting usually occurs according to a **vesting schedule**. You vest only while you work
   for the company.
   For example, it is very common to have stock or options vest over a period of four years,
   a bit at a time, where none of it is vested at first, and all of it is vested after four
   years.
-- 🄳 Vesting schedules can also have a **cliff**, where until you work for a given amount of
-  time, you are 0% vested.
-  For example, if your equity award had a one-year cliff and you only worked for the company
-  for 11 months, you would not get anything, since you have not vested in any part of your
-  award. Similarly, if the company is sold within a year, depending on what your paperwork
-  says, you may also receive nothing on the sale of the company.
-- A very common vesting schedule is vesting over **4 years**, with a **1 year** cliff.
-  This means you get 0% vesting for the first 12 months, 25% vesting at the 12th month, and
-  1/48th (2.08%) more vesting each month until the 48th month.
-  For example, if you leave just before a year is up, you get nothing, but if you leave
-  after 3 years, you get 75%.
-- 🄳 Vesting *can* be triggered outside of these timelines You may have
-  [**acceleration**](http://www.danshapiro.com/blog/2012/04/vesting-is-a-hack/), where vesting is
-  triggered if a company is sold (**single trigger**) or if it’s sold and you’re fired (**double
-  trigger**).
-  - ☝️ Acceleration is common for founders and not so common for employees.
-    It’s worth understanding acceleration and triggers, but also may not be something you can
-    negotiate unless you are going to be in a key role.
-- 🌪 Cliffs are an important topic.
-  When they work well, cliffs are an effective and reasonably fair system to both employees
-  and companies.
-  But they also can be abused or lead to misunderstandings.
-  - The intention of a cliff is to make sure someone is serious about contributing to a
+- 🄳 Vesting schedules can also have a **cliff**, designating that you must work for a given
+  amount of time before you vest at all.
+
+  - For example, if your equity award had a one-year cliff and you only worked for the company
+    for 11 months, you would not get anything, since you haven’t vested in any part of your
+    award. Similarly, if the company is sold within a year, depending on what your paperwork
+    says, you may also receive nothing on the sale of the company.
+  - A very common vesting schedule is vesting over **4 years**, with a **1 year** cliff.
+    This means you get 0% vesting for the first 12 months, 25% vesting at the 12th month, and
+    1/48th (2.08%) more vesting each month until the 48th month.
+    If you leave just before a year is up, you get nothing, but if you leave after 3 years,
+    you get 75%.
+  - 🌪 Cliffs are an important topic.
```

Comment:
I think I'll move this back to the top. I don't think we can cover everything about cliffs as a sub-bullet of the definition. It's too big. Arguably we should have a whole subsection on cliffs.

Reply:
Maybe we do. It's not easy to be introduced to a concept and then learn more about it later on when you thought you'd already been told what's important. So if you move it down but we don't create a new subsection, just remember that transitional sentences can be lifesavers--and are needed a lot in this Guide, where information is spread out. Simple additions like, "We mentioned cliffs, and there are a few more details you should know..." These really help the reader not feel like they just missed some connection.